apps/mull-ui/src/app/pages/login/login.scss

```diff
@@ -4,97 +4,101 @@
   display: grid;
   grid-template-columns: 1fr;
   grid-row-gap: 1.5rem;
-}
 
-.login-mull-logo {
-  padding-top: 4rem;
-  display: block;
-  margin: 0 auto;
-  width: 30%;
-  padding-bottom: 3rem;
-}
+  .login-mull-logo {
+    padding-top: 4rem;
+    display: block;
+    margin: 0 auto;
+    width: 30%;
+    padding-bottom: 3rem;
+  }
 
-.facebook {
-  margin-bottom: 2rem;
-  background-color: $facebook;
-  padding-left: 1.2rem;
-}
+  .facebook {
+    margin-bottom: 2rem;
+    background-color: $facebook;
+    padding-left: 1.2rem;
+  }
 
-.custom-text-input-container,
-.facebook,
-.google,
-.login,
-.separator,
-.twitter {
-  align-self: center;
-  justify-self: center;
-}
+  .custom-text-input-container,
+  .facebook,
+  .google,
+  .login,
+  .separator,
+  .twitter {
+    align-self: center;
+    justify-self: center;
+  }
 
-.separator {
-  display: flex;
-  align-items: center;
-  color: $primary-color;
-  width: 100%;
-}
+  .custom-text-input-container {
+    margin-bottom: 1.5rem;
+  }
 
-.separator::before,
-.separator::after {
-  content: '';
-  flex: 1;
-  border-bottom: 1px solid $primary-color;
-}
+  .separator {
+    display: flex;
+    align-items: center;
+    color: $primary-color;
+    width: 100%;
+  }
 
-.separator::before {
-  margin-right: 0.25rem;
-}
-.separator::after {
-  margin-left: 0.25rem;
-}
+  .separator::before,
+  .separator::after {
+    content: '';
+    flex: 1;
+    border-bottom: 1px solid $primary-color;
+  }
 
-.login,
-.twitter,
-.google,
-.facebook {
-  width: 100%;
-  height: 2.8rem;
-  cursor: pointer;
-  border-radius: 16px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  flex-direction: row;
-}
+  .separator::before {
+    margin-right: 0.25rem;
+  }
+  .separator::after {
+    margin-left: 0.25rem;
+  }
 
-.twitter,
-.google,
-.facebook {
-  color: white;
-  font-weight: bold;
-}
+  .login,
+  .twitter,
+  .google,
+  .facebook {
+    width: 100%;
+    height: 2.8rem;
+    cursor: pointer;
+    border-radius: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
```

Comment:
default flex-direction is row, no need to specify it explicitly. If it's elsewhere please remove from there as well
```suggestion
```